### DISCUSSION TITLE
Can the decimal point be adjusted for voltage?

### DISCUSSION THREAD

**Discussion by BulvaDj** [syssi/esphome-atorch-dl24#84] [Q&A] [answered]
Hi,

trying to discharge a battery NICD cell, my DL24 shows me a voltage of for example 1.2854. Home assistant shows me 1.2V. If I add accuracy_decimals: 3 it shows 1.20V.

Is it possible to set it to show 1.28V please? Otherwise everything works perfectly.

Thank you and sorry for the bad English - translator...

**Comment by syssi** [comment, marked as answer]
No. This isn't possible because the payload of the bluetooth notification frame doesn't provide a better resolution. See https://github.com/syssi/esphome-atorch-dl24/blob/main/docs/protocol-design.md#dc-meter-report

**Reply by BulvaDj** [reply]
Thank you very much for your very quick reply, I understand now.
